doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -154,3 +272,133 @@ Here is the offline configuration tool workflow:
+
+- Follow the :ref:`instruction <getting-started-building>` to install the
+  ACRN hypervisor dependencies and tools on your development host.
+
+- Follow the `get acrn repo`_ to download ACRN hypervisor repo to your host.
+
+- Install ACRN config app dependencies:
+
+  .. code-block:: console
+
+     $ cd ~/acrn-hypervisor/misc/acrn-config/config_app
+     $ sudo pip3 install -r requirements
+
+How to use ACRN config app
+==========================
+#. Launch ACRN config app:
+
+   .. code-block:: console
+
+      $ python3 app.py
+
+#. The browser should be launched and navigated to the website:
+   `http://127.0.0.1:5001/` automatically, or you may need to visit this website manually.
+
+   .. note:: The ACRN config app is supported on Chrome, Firefox or MS Edge, do not use IE.
+
+   The website as shown below:
+
+   .. figure:: images/acrn_config_main_menu.png
+      :align: center
+      :scale: 80%
+      :name: ACRN config tool main menu
+
+#. Set the board info:
+
+   a. Click the button **Import Board info**.
+
+      .. figure:: images/click_import_board_info_button.png
+         :align: center
+         :scale: 80%
+
+   #. Upload the board info you have generated by ACRN config tool.
+
+   #. After board info uploaded, you will see the board name from the Board info list,
+      select the board name to be configured.
+
+      .. figure:: images/select_board_info.png
+         :align: center
+         :scale: 80%
+
+#. Choose a scenario from the “Scenario Setting” menu which lists all the scenarios
+   including default scenarios and user-defined scenarios for the board you selected
+   in the previous step. The scenario configuration xmls are located in
+   ``acrn-hypervisor/misc/xmls/config-xmls/[board]/``.
+
+   .. figure:: images/choose_scenario.png
+      :align: center
+      :scale: 80%
+
+#. It is also allowed to use a customized scenario xml by clicking the Import button.
+   The config app will auto direct to the new scenario xml once the import is completed.
+
+#. The configurable items will be displayed after one scenario is selected. Here is
+   the example of "SDC" scenario:
+
+   .. figure:: images/configure_scenario.png
+      :align: center
+      :scale: 80%
+
+   - You can editor those items directly in the text boxes, choose single or even multiple
+     items from the drop down list.
+
+   - Read-only items are marked as grey.
+
+   - Hover the mouse pointer over the item to display the description.
+
+#. Click **Export** button to save the scenario xml, you can rename it in the pop-up modal.
+
+   .. note:: All customized scenario xmls will be in user-defined groups which located in
+      ``acrn-hypervisor/misc/xmls/config-xmls/[board]/user_defined/``.
+
+   After the scenario is saved, the page will auto direct to the saved scenario xmls.
+   You can delete the configured scenario by click button **Export** -> **Remove**.
+   Before saving the scenario xml, the config app will validate the configurable items;
```

Comment:
suggest to organize this paragraph like this: Before saving, validate. After saving, direct to the saved scenario. You can delete...